src/ActionExecutor/UserAction.php

```diff
@@ -81,9 +81,9 @@ class UserAction extends Modal implements Interface_, jsInterface_
     /**
      * A form for action argument and fields user entry.
      *
-     * @var string
+     * @var array
      */
-    public $form = Form::class;
+    public $form = [Form::class];
```

Comment:
At least in the code comments we should mention it is a seed ...

Reply:
I expect much more inconsistencies, worth to fix consistently at once.